app/javascript/packs/admin.js

```diff
@@ -11,8 +11,8 @@ require('magnific-popup')
 require('datatables.net-bs')
 require('bootstrap-tagsinput')
 // files
-//require('admin/theme')
+require('admin/theme')
 require('admin/sidebar')
 require('admin/add_book')
-require('admin/datatable_book')
 require('admin/users_table')
+require('admin/table_with_details')

```

Comment:
I don't think this is a good name as well. `books_table` is fine

Reply:
it isn't for book only it's for author and category too